wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -156,7 +177,7 @@ def save_pdf(self, response):
         @returns requests 0 0
         """
 
-        is_pdf = response.headers.get('content-type', '') == b'application/pdf'
+        is_pdf = b'application/pdf' in response.headers.get('content-type', '')
```

Comment:
why did we switch to approximate match?

Reply:
Because of the Who new headers which doesn't always return a simple 'application/pdf', but sometimes add things after. The check remains the same for the Nice website

Reply:
this can be dangerous but in this context i think it is unlikely it will cause a problem. 🤞

Reply:
From what I saw, the header only (sometimes, and sometimes not) includes the charset used in the pdf. Would it be better if we checked the header value against a list of "enabled" headers?
Something like:
```python
    enabled_headers = [b'application/pdf', b'application/pdf;charset=ISO-8859-1' ...]
    if response.headers.get('content-type', '') in enabled_headers:
        ...
```

Reply:
without being an expert it seems that `;` is the separator in the specification so why not split the string and strip the tokens and check whether 'application/pdf' is in the list of tokens.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

Reply:
... I don't exactly know how I didn't think about that, but yep, I'll will definitely do that 😄